src/main/java/com/google/sps/servlets/ClassObject.java

```diff
@@ -0,0 +1,19 @@
+package com.google.sps.servlets;
+
+public class ClassObject {
```

Comment:
Looking at how this is used you could probably use a constructor here.

Reply:
Also, class is an ambiguous word, in your design doc you have course/term instead. Please change here and below

Reply:
I changed every reference to Course to make it more easily readable :)